tensorboard/webapp/widgets/line_chart_v2/lib/drawable.ts

```diff
@@ -23,15 +23,26 @@ import {
 import {PaintBrush} from './paint_brush';
 import {ObjectRenderer} from './renderer/renderer_types';
 
-class RenderCacheContainer {
-  private prevFrameCache = new Map<string, any>();
-  private currFrameCache = new Map<string, any>();
+type Cacheable = {};
 
-  getFromPreviousFrame(key: string): any {
-    return this.prevFrameCache.get(key);
+export interface RenderCachePublicApi {
+  getFromPreviousFrame(key: string): Cacheable | null;
+  setToCurrentFrame(key: string, value: Cacheable): void;
+}
```

Comment:
Would it be cleaner to move this interface definition into `paint_brush.ts` or another common file, to avoid a circular dependency between `drawable.ts` and `paint_brush.ts`?